core/util/src/main/scala/net/liftweb/util/BaseField.scala

```diff
@@ -113,6 +114,14 @@ trait SettableField extends ReadableField with SettableValueHolder {
    */
   def toForm: Box[NodeSeq]
 
+  protected def html5RequiredAttribute: MetaData = {
```

Comment:
Only thing I would say is that, since making something `protected` makes it part of public API, we may want to consider making this a little more generic. Perhaps `additionalHtmlAttributes`?

Reply:
That sounds useful. But yeah, good catch. I'll try and see if there's a way I can avoid adding to the public API.

Reply:
Unless you want to add that method to the public API, how about making it `private [liftweb]` in some companion object. For ex. as a method in `SettableField` where the `required` attribute is defined.

Reply:
I think starting as `private[liftweb]` is fine. If we want it to be public API we can adjust later. Can't take that back :)